docs/en/reference/index.rst

```diff
@@ -132,3 +132,15 @@ need to execute PHP_CodeSniffer with the tests folder and ensure it matches the
 .. code-block:: bash
 
     $ ./vendor/bin/phpcs tests/input --report=summary --report-file=phpcs.log; diff tests/expected_report.txt phpcs.log
+
+If you have implemented custom sniff also run PHPStan and and PHPUnit.
```

Comment:
```suggestion
If you have implemented custom sniff also run PHPStan and PHPUnit.
```

Reply:
@greg0ire Please, comment in the original PR :)

Reply:
woops